src/main/resources/templates/partials/navbar.ftl

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+  <!-- Brand -->
+  <a class="navbar-brand" href="/home">
+    <img src="img/UCSB.png"  height="32" class="d-inline-block align-top" alt="">
+    Q&A
+  </a>
+
+  <!-- Links -->
+  <div class="navbar-nav">
+    <a class="nav-item nav-link" href="/home">Home</a>
+    <a class="nav-item nav-link" href="/questions">Questions</a>
+    <a class="nav-item nav-link" href="/profile">Profile</a>
+  </div>
+
+  <!-- Search Bar -->
+  <form class="form-inline ml-auto">
+    <input class="form-control mr-sm-2" type="search" placeholder="Search..." aria-label="Search">
+    <button class="btn btn-success my-2 my-sm-0" type="submit"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAAZCAYAAADE6YVjAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAGJSURBVEhL5ZTBKkRhGIZPQ8RSE3sKZ4WysJKyVW5hElcgspCSuAOThZlhOQs2cgGaKRcgGsmWFUIaUWc83+klNZz/F7OYPPV0/v/73+98M6eZE/xParVaaxRFk7jGeodrFhcwVOR3cKNpvODmMawfsKqt7Q+xT/GfQ/MKGjYkw7VLR3bWj+v4iLc4riN/aJrTJ93DDpXrIDLI+SXese5V2Q3hNE33WGbdpvK3kAvxCQ9UcsONl9C+xahKTshuqGdApWQIlvBMWy/Ihxoyr1IyBK+xqK039DxjVttkCN5gQVtv6LFfWl7bZAieYklbL3hSaT2uVZWSIbiJL9ijkhOyszYEJlRKhoYRS3P1er5EO8naf6XCukVlNzTk0JhR6Us4b8f9OBlFUyq74dOkaMjbtzFYb2G3jj+gNobHyiyr7Ia8DcipsYDbaLxiGXexiOfK2Osko3Y3hD8PsGtK9WH29mM4QXsZXuERLnKWjpt9sBvSVDfgz2j+AQY3HsJqwwa8Y4MaOqDJCYI3BNH/UsWSmtwAAAAASUVORK5CYII="></button>
```

Comment:
It would be great if we could get these images into files and reference them instead of having base64 encoded images in the HTML. It would make it easier to read.